pkg/resources/metrics.go

```diff
@@ -503,15 +503,15 @@ func reconcilePostgresFreeableMemoryAlert(ctx context.Context, client k8sclient.
 	// build and reconcile postgres low freeable memory alert
 	alertName := "PostgresFreeableMemoryLow"
 	ruleName := "postgres-freeable-memory-low"
-	alertDescription := "The postgres instance {{ $labels.instanceID }} for product {{  $labels.productName  }}, freeable memory is currently under 20 percent of its capacity"
+	alertDescription := "The postgres instance {{ $labels.instanceID }} for product {{  $labels.productName  }}, freeable memory is currently under 5 percent of its capacity"
 	labels := map[string]string{
 		"severity": "critical",
 		"product":  installationName,
 	}
 
 	// checking if the percentage of freeable memory is less than 5% of the max memory
 	// conversion formula is MiB = bytes / (1024^2)
-	alertExp := intstr.FromString("(cro_postgres_freeable_memory_average) < ((cro_postgres_max_memory / 100 ) * 5)")
+	alertExp := intstr.FromString("(cro_postgres_freeable_memory_average) < on (instanceID) ((cro_postgres_max_memory / 100 ) * 5)")
```

Comment:
I had a look at what this change does all be it at a higher percentage(55%). It makes the alert work very differently. Not sure if this is a good idea 

<img width="1887" height="879" alt="image" src="https://github.com/user-attachments/assets/f93ea890-63f6-43ed-af64-0cc9758c9f8a" />

I think we should revert this change.

Reply:
done. Thank you for review and details provided.